cypress/integration/table.feature

```diff
@@ -0,0 +1,149 @@
+Feature: Table component
+  I want to change Table component properties
+
+  Background: Open Table component page
+    Given I open "Table" component page
+
+  @positive
+  Scenario Outline: Page size records is set to <pageSizeRecords>
+    When I set pageSize to "<pageSizeRecords>"
+    Then I see <pageSizeRecords> records
+    Examples:
+      | pageSizeRecords |
+      | 0               |
+      | 1               |
+      | 2               |
+      | 5               |
+
+  @ignore
+  @negative
+  # ignored because not working correctly during development
+  Scenario Outline: Page size records is set out of scope <pageSizeRecords>
+    When I set pageSize to "<pageSizeRecords>"
+    Then I see 0 records
+    Examples:
+      | pageSizeRecords         |
+      | -1                      |
+      | -10                     |
+      | Sample text             |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      | <>                      |
+
+  @positive
+  Scenario: I enable selectable
+    When I check selectable checkbox
+    Then rows are selectable
+
+  @positive
+  Scenario: I disable selectable
+    When I check selectable checkbox
+      And I uncheck selectable checkbox
+    Then rows are not selectable
+
+  @positive
+  Scenario Outline: Row <rowNumber> is highlighted
+    When I check highlightable checkbox
+      And row number <rowNumber> is clickable
+      And I click row by number <rowNumber>
+    Then row number <rowNumber> is highlighted
+    Examples:
+      | rowNumber |
+      | 1         |
+      | 2         |
+      | 3         |
+
+  @positive
+  Scenario Outline: Row <rowNumber> is not highlighted
+    When I check highlightable checkbox
+      And I uncheck highlightable checkbox
+      And row number <rowNumber> is not clickable
+      And I click row by number <rowNumber>
+    Then row number <rowNumber> is not highlighted
+    Examples:
+      | rowNumber |
+      | 1         |
+      | 2         |
+      | 3         |
+
+  @ignore
+  @positive
+  #ignored because it is not working correctly during development
+  Scenario: Enable shrink property
+    When I check shrink checkbox
+    Then row is shrinked
+
+  @ignore
+  @positive
```

Comment:
should be a comment here too -> #ignored because it is not working correctly during development

Reply:
fixed